cdap-api/src/main/java/co/cask/cdap/api/metadata/MetadataReaderContext.java

```diff
@@ -25,13 +25,15 @@
   /**
    * Returns a Map of {@link MetadataScope} to {@link Metadata} representing all metadata (including properties
    * and tags) for the specified {@link MetadataEntity} in both {@link MetadataScope#USER} and
-   * {@link MetadataScope#SYSTEM}.
+   * {@link MetadataScope#SYSTEM}. The map will be empty if the there is no metadata associated with the given
+   * metadataEntity.
    */
   Map<MetadataScope, Metadata> getMetadata(MetadataEntity metadataEntity);
 
   /**
    * Returns a {@link Metadata} representing all metadata (including properties and tags) for the specified
-   * {@link MetadataEntity} in the specified {@link MetadataScope}.
+   * {@link MetadataEntity} in the specified {@link MetadataScope}. The map will be empty if the there is no metadata
```

Comment:
nit: This does not return map, but return an instance of Metadata with empty properties and tags right?